dataprofiler/data_readers/data_utils.py

```diff
@@ -15,10 +17,10 @@
 from .. import dp_logging
 from .filepath_or_buffer import FileOrBufferHandler, is_stream_buffer  # NOQA
 
-logger = dp_logging.get_child_logger(__name__)
+logger: Logger = dp_logging.get_child_logger(__name__)
```

Comment:
if you import `annotations` from `__futures__` can you not import the `Logger` here?